packages/fxa-content-server/app/scripts/templates/partial/account-suggestion.mustache

```diff
@@ -1,6 +1,12 @@
 {{#showAccountSuggestion}}
-  <div class="info mx-auto mb-5" id="suggest-account">
+  <div class="info" id="suggest-account">
+    <p class="info-text">
       {{#unsafeTranslate}}Why do I need to create this account? <a %(escapedSuggestionAttrs)s>Find out here</a>{{/unsafeTranslate}}
-      <span class="dismiss" tabindex="-1">&#10005;</span>
+    </p>
+    <button type="button" class="dismiss" aria-label="Close">
```

Comment:
Q: Should "Close" be localized?

Reply:
Agreed, I've added in the tags to localize the aria-label for both banners and the tooltip.